etc/a.gwf

```diff
@@ -243,6 +275,12 @@ max_cousins=2000
 # Maximum level of displayed cousins
 max_cousins_level=5
 
+# Manage the behaviour of the template command %has_events;
+# never -> returns false
+# always -> returns true 
+# _ -> returns true if there are events beyond birth, baptism, marriage, death, burial
+#has_events
```

Comment:
I was not aware of this, and I will certainly drop its support. Unless someone can provide a use case where this is useful?

Reply:
Not essential. Would have been convenient to avoid repeating in a simplified timeline events already displayed in the personal header.